ramls/schemas/profiles/transformations.json

```diff
@@ -15,7 +15,7 @@
     },
     "enabled": {
       "description": "Indicates whether field should be mapped",
-      "type": "string"
+      "type": "boolean"
```

Comment:
I think also adding default false would make sense

Reply:
Thanks, it's good point, I will add default false